src/components/SearchModal/BlockedToken.tsx

```diff
@@ -0,0 +1,71 @@
+import { Trans } from '@lingui/macro'
+import { Token } from '@uniswap/sdk-core'
+import { ButtonPrimary } from 'components/Button'
+import { AlertCircle, ArrowLeft } from 'react-feather'
+import styled from 'styled-components/macro'
+import { CloseIcon, TYPE } from 'theme'
+
+import TokenImportCard from './TokenImportCard'
+
+const Wrapper = styled.div`
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  flex: 1 1 auto;
+  height: 100%;
+  width: 100%;
+`
+const Button = styled(ButtonPrimary)`
+  margin-top: 1em;
+  padding: 10px 1em;
+`
+const Content = styled.div`
+  padding: 1em;
+`
+const Copy = styled(TYPE.body)`
+  text-align: center;
+  margin: 0 2em 1em !important;
+  font-weight: 400;
+  font-size: 16px;
+`
+const Header = styled.div`
+  align-items: center;
+  display: flex;
+  gap: 14px;
+  justify-content: space-between;
+  padding: 20px;
+  width: 100%;
+`
+const Icon = styled(AlertCircle)`
+  stroke: ${({ theme }) => theme.text2};
+  width: 48px;
+  height: 48px;
+`
+interface BlockedTokenProps {
+  onBack: (() => void) | undefined
+  onDismiss: (() => void) | undefined
+  blockedTokens: Token[]
+}
+
+const BlockedToken = ({ onBack, onDismiss, blockedTokens }: BlockedTokenProps) => (
+  <Wrapper>
+    <Header>
+      {onBack ? <ArrowLeft style={{ cursor: 'pointer' }} onClick={onBack} /> : <div />}
+      <TYPE.mediumHeader>
```

Comment:
This components seems like something new we could use? 
Are we using it in this version?

Reply:
Yep, this is new modal for blocked tokens https://raw.githubusercontent.com/The-Blockchain-Association/sec-notice-list/master/ba-sec-list.json. Added

![BlockedToken](https://user-images.githubusercontent.com/34926005/146095406-82dc634d-02a2-4ab2-95b6-a31f244d7994.jpeg)